book/todo.md

```diff
@@ -17,21 +17,22 @@ I am editing the chapters following the first iteration of the course.
 - [x] Debugging tips have moved out of the [Preliminaries](preliminaries.md)
 - [x] Clipping moved from [Chapter 10](reflow.md) to [Chapter 2](graphics.md) 
 - [ ] The [Preliminaries](preliminaries.md) should install Python
-- [ ] [Chapter 3](text.md) should implement multiple text sizes
-- [ ] [Chapter 4](html.md) needs to motivate attributes
+- [x] [Chapter 3](text.md) should implement multiple text sizes
+- [x] [Chapter 4](html.md) needs to motivate attributes
+- [ ] [Chapter 5](layout.md)
 - [ ] [Chapter 7](chrome.md) should add forward buttons
-- [ ] [Chapter 10](reflow.md) needs a rewrite to handle a major bug
+- [ ] [Chapter 8](forms.md) needs minor edits
+- [ ] [Chapter 10](reflow.md) needs to fix a major bug
```

Comment:
Is the bug explained somewhere? Write an issue on emberfox and link to it?

Reply:
The bug is now issue #8, and means that reflow does not actually satisfy the reflow invariant when heights change!